cypress/integration/team/teams_spec.js

```diff
@@ -107,4 +107,45 @@ describe('Teams Suite', () => {
         // 9. Remove user from team
         cy.removeTeamMember(teamURL, user.firstName);
     });
+
+    it('TS14633 Leave all teams', () => {
+        // 1. Login and go to /
+        cy.login('user-1');
```

Comment:
The test meets the spec requirement as specified in the ticket.  However, this will potentially break for succeeding tests since test "user-1" is no longer a member of any team.

Options could be used of test user other than "user-1" or used "user-1" but re-join the teams after completing this test.